index.js

```diff
@@ -1 +1,224 @@
-console.log('Live reloading')
\ No newline at end of file
+const startButton = document.querySelector('.startButton');
+
+const restart = document.querySelector(".restartButton");    // US7 Button.
+
+const redButton = document.querySelector('.redB');
+const greenButton = document.querySelector('.greenB');
+const blueButton = document.querySelector('.blueB');
+const yellowButton = document.querySelector('.yellowB');
+
+const textLevel = document.querySelector('.levelLabel');
+
+let gameSeqnc = [];
+let playerSeqnc = [];
+let round = 1;
+const MAX_ROUNDS = 20;
+let hardmode = false;
```

Comment:
So Gabriel, I don't think that this loc is doing anything, can you tell me more about it? if not we need to delete it.

Reply:
Foo